tests/ut/azure_iot_hub_client_properties_ut.c

```diff
@@ -317,12 +344,53 @@ static void testAzureIoTHubClientProperties_GetPropertiesVersion_Success( void *
     assert_int_equal( ulVersion, 5 );
 }
 
+static void testAzureIoTHubClientProperties_GetPropertiesVersionGetDocument_Success( void ** ppvState )
+{
+    AzureIoTHubClient_t xTestIoTHubClient;
+    AzureIoTJSONReader_t xJSONReader;
+    AzureIoTJSONTokenType_t xTokenType;
+    AzureIoTHubMessageType_t xResponseType = eAzureIoTHubPropertiesGetMessage;
+    uint32_t ulVersion;
+
+    assert_int_equal( AzureIoTJSONReader_Init(
+                          &xJSONReader,
```

Comment:
nit: formatting.

Reply:
yea this is just uncrustify doing its thing